### PR TITLE
clean view of CMS Article Categories #7072

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

**Comment by aroraumang** [comment]
LGTM

File: view/cms_article_category_form.xml [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
+    <group colspan="3" col="4" id="active" >
+	<field name="unique_name" colspan="2" />
+        <label name="active" />
+        <field name="active" />
+    </group>
+    <notebook colspan="4">
+        <page id="articles" string="Articles">
+            <field name="articles" colspan="4" />
+        </page>
+        <page  id="sepr_desc" string="Description">
```
Comment (aroraumang) [inline view/cms_article_category_form.xml]: id="description"

File: view/cms_article_category_form.xml [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
+            <field name="description" colspan="4" />
+        </page>
+        <page id="sepr_articles1" string="Published Articles" >
```
Comment (aroraumang) [inline view/cms_article_category_form.xml]: id="published_articles"

File: view/cms_article_category_form.xml [hunk 3]
```diff
@@ -1,27 +1,32 @@
...
+            <field name="published_articles" colspan="4"/>
+        </page>
+        <page id="d_settings" string="Display Settings">
```
Comment (aroraumang) [inline view/cms_article_category_form.xml]: id="display_settings"

**Comment by vishu1993** [comment]
cms article

**Comment by vishu1993** [comment]
updated cms article

**Comment by aroraumang** [comment]
LGTM

**Comment by aroraumang** [comment]
LGTM

File: view/cms_article_category_form.xml [hunk 4]
```diff
@@ -1,27 +1,32 @@
...
-        colspan="4"
-        string="Description"
-        id="sepr_desc"/>
-    <field name="description" colspan="4" />
-    <separator
-        colspan="4"
-        string="Published Articles"
-        id="sepr_articles1"/>
-    <field name="published_articles" colspan="4"/>
+    <group colspan="3" col="4" id="active" >
```
Comment (prakashpp) [inline view/cms_article_category_form.xml]: `colspan=4`

**Comment by vishu1993** [comment]
cleaned nereid cms

**Comment by vishu1993** [comment]
cmsarticlecategoriesview

**Pull request merged**